net/upnp/src/etc/inc/plugins.inc.d/miniupnpd.inc

```diff
@@ -226,6 +230,16 @@ function miniupnpd_configure_do($verbose = false)
             $config_text .= "min_lifetime=120\n";
             $config_text .= "max_lifetime=86400\n";
 
+            /* configure STUN server if needed */
+            if (!empty($upnp_config['stun_host']) && empty($upnp_config['overridewanip'])) {
+                if(empty($upnp_config['stun_port'])) {
+                    $upnp_config['stun_port'] == '3478';
```

Comment:
wouldn't do this if you use `??` underneath... which is safer than tainting configuration data (it could end up being saved)